src/OrchardCore.Modules/OrchardCore.Settings/Views/Admin/Culture.cshtml

```diff
@@ -53,3 +63,4 @@
     </table>
 </fieldset>
 
+@Html.ActionLink(T["Close"].Value, "Index", "Admin", new { area = "OrchardCore.Settings", groupId = "general" }, new { @class = "btn btn-primary" })
```

Comment:
`ActionLink(T["Close"].Value` no or there can be double encoding. Don't pass a translation to a helper that only accepts a string. Use the `<a>` tag directly in this case. and `ActionUrl`